release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,225 @@
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed, refer to the table below. Additional information about some removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+- GA: General Availability
+- TP: Technology Preview
+- DEP: Deprecated
+- REM: Removed
+
+.Deprecated and removed features tracker
+[cols="3,1,1,1",options="header"]
+|====
+|Feature |{product-title-short} 3.69 |{product-title-short} 3.70 | {product-title-short} 3.71
+
+|Ability to add comments to alerts and processes
+|DEP
+|DEP
+|REM
+
+|Anchore, Tenable, and Docker Trusted registry integrations
+|DEP
+|DEP
+|REM
+
+|External authorization plug-in for scoped access control
+|DEP
+|DEP
+|REM
+
+|`FROM` option in the Disallowed Dockerfile line policy field
+|GA
+|DEP
+|DEP
+
+| `PodSecurityPolicy` (PSP) Kubernetes objects
+| GA
+| DEP
+| REM
+
+|`RenamePolicyCategory` and `DeletePolicyCategory` API endpoints
+|GA
+|DEP
+|DEP
+
+| `--rhacs` option for the `roxctl` helm output command
+| DEP
+| DEP
+| REM
+
+|`/v1/nodes` and `/v1/images` API endpoint response: `firstNodeOccurrence` response body parameter
+|DEP
+|DEP
+|REM
+
+
+| Permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, `User`, `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, `Risk`, `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, `SignatureIntegration`, `ImageComponent`
+| GA
+| GA
+| DEP
+
+| Retrieving groups by property
+| GA
+| GA
+| DEP
+
+|====
+
+[id="deprecated-features-371"]
+=== Deprecated features
+
+* Permissions for permission sets will be grouped for simplification. The following list describes the new permissions and indicates the deprecated permissions that will be removed in a future release:
+** New permission Access will replace the permissions `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, and `User`.
+** New permission `DeploymentExtension` will replace the permissions `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, and `Risk`.
+** New permission Integration will deprecate the permissions `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, and `SignatureIntegration`.
+** New permission Image will replace the permission `ImageComponent`.
+* Previously, groups were retrieved by the field props (`props.authProviderId`, `props.key`, `props.value`). This field will be replaced by the new `props.id` field. Use the props.id field to retrieve groups in the Application Programming Interface (API). Note the following:
+** Retrieval via the `props` fields will be removed in version {product-title-short} version 3.73.
```

Comment:
s/ via / thorugh (or by) https://docs.engineering.redhat.com/display/CCSENABLE/Avoid+these+words

We also try not to predict the future. There are a lot of ramifications for this. It would be better to say "in a future version of {product-title-short}. They are vary adamant about not saying "X will be removed in *this* release".

Reply:
I've had that discussion with them too about specific releases, but they have pushed back. I'll see if I can change it.